resources/lib/SubsceneUtilities.py

```diff
@@ -77,23 +84,39 @@ def get_episode_pattern(episode):
 
 def log(module, msg):
     global subscene_start
-    xbmc.log((u"### [%s] %f - %s" % (module, time.time() - subscene_start, msg,)).encode('utf-8'), level=xbmc.LOGDEBUG)
+    loglevel=xbmc.LOGDEBUG
+#    loglevel=xbmc.LOGWARNING
+    if sys.version_info.major == 3:
+        xbmc.log("### [%s] %f - %s" % (module, time.time() - subscene_start, msg,), level=loglevel)
+    else:
+        xbmc.log((u"### [%s] %f - %s" % (module, time.time() - subscene_start, msg,)).encode('utf-8'), level=loglevel)
```

Comment:
Doesn't this line also work well with Python 3?

Reply:
no. xbmc.log in Kodi 19 requires a string and not a byte object which is what you get when encoding the string. https://alwinesch.github.io/group__python__xbmc.html